src/functions.php

```diff
@@ -311,11 +311,11 @@ function filterMatches($attribute, $filterValues): bool
  *
  * @param array $cards
  * @param array $conversions
- * @return SimpleXMLElement
+ * @return SimpleXMLElement     the XML phone book in Fritz Box format
  */
 function export(array $cards, array $conversions): SimpleXMLElement
```

Comment:
either rename this to `exportPhonebook` or (what I would prefer) undo renaming the other functions. As-is its inconsistent.

Reply:
ok - went for exportPhonebook